Dockerfile

```diff
@@ -1,8 +1,13 @@
-FROM alpine:latest
+FROM alpine:3.5
 MAINTAINER Yannick Buron yburon@goclouder.net
 
 RUN apk add --update ssmtp wget curl rsync bash
```

Comment:
We have a lint failing here for using both curl & wget - clouder-community/clouder#194

I will disable the lint here